src/routes/examples/raster_source/+page.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  import MapLibre from '$lib/MapLibre.svelte';
+  import RasterTileSource from '$lib/RasterTileSource.svelte';
+  import RasterLayer from '$lib/RasterLayer.svelte';
+  import { mapClasses } from '../styles.js';
+  import code from './+page.svelte?raw';
+  import CodeSample from '$site/CodeSample.svelte';
+  import LineLayer from '$lib/LineLayer.svelte';
+
+  let pollutant = 'PM25_viridis';
+  $: url = `https://ukair.maps.rcdo.co.uk/ukairserver/services/aq_amb_2021/${pollutant}/MapServer/WMSServer?bbox={bbox-epsg-3857}&request=GetMap&version=1.3.0&format=image%2Fpng&crs=EPSG%3A3857&width=256&height=256&styles=&layers=20`;
```

Comment:
It took me about an hour to figure this out; I think this is the minimal example of how to use WMS >_<